build.gradle

```diff
@@ -13,8 +13,10 @@ plugins {
 
 group = 'org.gradle.android'
 version = layout.projectDirectory.file("release/version.txt").asFile.text.trim()
+description = 'Gradle plugin to fix Android caching problems'
```

Comment:
For consistency with our other plugins, I would say `A Gradle plugin...'.

Reply:
Done